### PR TITLE
Add README, CONTRIBUTING and issue and pull request templates

### PR DESCRIPTION
**Pull request by MrSquaare**

### CODE REVIEW

**Review by Madfish5415 — COMMENTED** [review]
Do we keep the `type:` in front of the labels?

**Comment by MrSquaare** [comment]
> Do we keep the `type:` in front of the labels?

I think they need to be removed

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Feature
+about: 'Request a feature'
+title: 'Feature: '
+labels: 'enhancement'
```
Comment (MrSquaare) [inline .github/ISSUE_TEMPLATE/feature_template.md]: Must use the label 'feature' instead of 'enhancement'

**Review by ColColty — DISMISSED** [review]

**Review by Madfish5415 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
-# template
\ No newline at end of file
+# template
+
+Short description of the project
+
```
Comment (MrSquaare) [inline README.md:4]: Missing Table of Contents

**Review by ColColty — DISMISSED** [review]

**Review by ColColty — APPROVED** [review]

**Pull request merged**